server/api/users/users.test.ts

```diff
@@ -24,16 +26,27 @@ describe('auth router', () => {
 
       const res = await server.exec.get(`/api/users/${user.id}`);
       expect(res.status).toBe(200);
-      expect(res.body).toEqual({
-        id: user.id,
-        discord_user_id: user.discord_user_id,
-        discord_name: user.discord_name,
-        bio: user.bio,
-        twitter_username: user.twitter_username,
-        linkedin_url: user.linkedin_url,
-        github_username: user.github_username,
-        website: user.website,
-      });
+      expect(res.body).toEqual(getExpectedUserObject(user));
+      // Delete the user that was created
+      await User.destroy({ where: {}});
```

Comment:
Needed to delete User DB here because it was carrying over into the next test

Reply:
I think it will be better to instead have the test suite spin up a new test app before every test, rather than before all tests. This solution may not work if, say, we need to test that route more, or if the User object gets some related models (because then we'd have to delete them all)